src/lxc/monitor.c

```diff
@@ -205,7 +204,7 @@ int lxc_monitor_open(const char *lxcpath)
 	}
 
 	len = strlen(&addr.sun_path[1]) + 1;
```

Comment:
```c
int snprintf(char *str, size_t size, const char *format, ...);
```
`snprintf` will write at most **size** bytes (including the terminating null byte ('\0')) to **str**, so the  `addr.sun_path` awlays have a `null byte` and the end.